mono/mini/mini-exceptions.c

```diff
@@ -2172,7 +2172,7 @@ handle_exception_first_pass (MonoContext *ctx, MonoObject *obj, gint32 *out_filt
 					 mono_method_full_name (method, TRUE));
 		}
 
-		if (method->wrapper_type != MONO_WRAPPER_RUNTIME_INVOKE && mono_ex) {
+		if (method->wrapper_type != MONO_WRAPPER_RUNTIME_INVOKE && method->wrapper_type != MONO_WRAPPER_OTHER && mono_ex) {
```

Comment:
@vargaz What do you think about this ? Should we ignore these internal frames in the stacktrace like done here, or would you rather leave them and change/disable the tests ?

Reply:
I think the frames should show up in textual stack traces because they are useful, but don't show up in StackTrace/StackFrame.